openid-4-verifiable-presentations-1_0.md

```diff
@@ -2025,19 +2025,75 @@ The Verifier MAY send all the OpenID4VP request parameters as members in the req
 
 The Verifier MAY send a signed request, for example, when identification and authentication of the Verifier is required.
 
+The signed request allows the Wallet to authenticate the Verifier using one or more trust framework(s) other than the Web PKI utilized by the browser. An example of such a trust framework is the Verifier (RP) management infrastructure set up in the context of the eIDAS regulation in the European Union, in which case, the Wallet can no longer rely only on the web origin of the Verifier. This web origin MAY still be used to further strengthen the security of the flow. The external trust framework could, for example, map the Client Identifier to registered web origins.
```

Comment:
How does the signed request enable the use of trust frameworks representation using OpenID Federation?  (It may already do that, but more details to that effect would be useful to implementers.)

Reply:
The PR does not change the way client id schemes work. Consequently, all definitions for federation still hold. true.